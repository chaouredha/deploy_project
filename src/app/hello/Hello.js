import React from 'react'
import PropTypes from 'prop-types'

const Hello = ({ message }) => {
  return <div>{message}</div>
}

export default Hello

Hello.propTypes = {
  message: PropTypes.string
}
