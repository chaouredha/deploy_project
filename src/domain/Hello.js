import PropTypes from 'prop-types'

class Hello {
  constructor(message) {
    this.message = message
  }
}

Hello.propTypes = PropTypes.shape({
  message: PropTypes.string.isRequired
})

export default Hello
