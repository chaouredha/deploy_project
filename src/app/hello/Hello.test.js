import React from 'react'
import { render } from '@testing-library/react'
import { describe, it } from '@jest/globals'
import Hello from 'app/hello/Hello'

describe('Hello world', () => {
  it('displays Hello world', () => {
    const { asFragment } = render(<Hello message={'Test Message'}>contenu</Hello>)

    expect(asFragment()).toMatchSnapshot()
  })
})
