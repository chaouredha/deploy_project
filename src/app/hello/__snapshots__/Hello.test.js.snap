// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello world displays Hello world 1`] = `
<DocumentFragment>
  <div>
    Test Message
  </div>
</DocumentFragment>
`;
